docs/user-guide/timepix_images.ipynb

```diff
@@ -0,0 +1,425 @@
+    "\n",
+    "        self._figure = figure\n",
+    "        self._tool = Rectangles(\n",
+    "            ax=self._figure.ax, autostart=kwargs.get('autostart', False)\n",
+    "        )\n",
+    "        self.rectangles = []\n",
+    "        self._draw_node = pp.Node(lambda: self.rectangles)  # Empty rectangle info.\n",
+    "        self._destination = destination\n",
+    "        self._destination.add_parents(self._draw_node)\n",
+    "        self._tool.on_create(self.update_node)\n",
+    "        self._tool.on_vertex_release(self.update_node)\n",
+    "        self._tool.on_drag_release(self.update_node)\n",
+    "        self._tool.on_remove(self.update_node)\n",
+    "\n",
+    "    def update_node(self, _):\n",
+    "        self.rectangles = [\n",
+    "            _get_rect_info(artist=artist, figure=self._figure)\n",
+    "            for artist in self._tool.children\n",
+    "        ]\n",
+    "        self._draw_node.func = lambda: self.rectangles\n",
+    "        self._draw_node.notify_children(\"\")  # Empty message.\n",
+    "\n",
+    "    def start_stop(self):\n",
+    "        \"\"\"\n",
+    "        Toggle start or stop of the tool.\n",
+    "        \"\"\"\n",
+    "        if self.value:\n",
+    "            self._tool.start()\n",
+    "        else:\n",
+    "            self._tool.stop()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Prepare the merge method\n",
+    "from functools import reduce\n",
+    "\n",
+    "\n",
+    "def _show_histogram(da: sc.DataArray, rect_infos: list[RectInfo]) -> sc.DataArray:\n",
+    "    if len(rect_infos) > 0:\n",
+    "        masks = [\n",
+    "            (normalized_img.coords['x'] > rect_info.max_x)\n",
+    "            | (normalized_img.coords['x'] < rect_info.min_x)\n",
+    "            | (normalized_img.coords['y'] > rect_info.max_y)\n",
+    "            | (normalized_img.coords['y'] < rect_info.min_y)\n",
+    "            for rect_info in rect_infos\n",
+    "        ]\n",
+    "        mask = reduce(lambda x, y: x & y, masks)\n",
+    "        selected_sample_region = da.copy(deep=False)\n",
+    "        selected_sample_region.masks['roi'] = mask['x', :-1]['y', :-1]\n",
+    "        return selected_sample_region.sum('x').sum('y')\n",
+    "    else:\n",
+    "        return da.sum('x').sum('y')\n",
+    "\n",
+    "\n",
+    "original_data_node = pp.Node(normalized_img)\n",
+    "merge_node = pp.Node(_show_histogram)\n",
+    "merge_node.add_parents(original_data_node)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_node = pp.Node(normalized_img.sum('wavelength'))\n",
+    "f2d = pp.imagefigure(\n",
+    "    data_node, norm='log', title=\"Region of Interest Selection\", cbar=True\n",
+    ")\n",
+    "\n",
+    "roi_merge_tool = MergedRectanglesTool(\n",
```

Comment:
See https://github.com/scipp/plopp/pull/396#discussion_r1865787625
I think we can do without creating a new tool?